libraries/botbuilder-dialogs-adaptive/tests/tsconfig.json

```diff
@@ -1,9 +1,11 @@
 {
-  "extends": "../tsconfig.json",
```

Comment:
Note: New or new changes to `.ts` tests weren't actually running because the test files were getting built in `\lib`. 🤦‍♂️

Reply:
Oi - good find!

Reply:
This sort of thing makes me really want to:

1. Nothing should have any test items as dependencies, then
2. `--require ts-node/register`

Reply:
I think you still need a `tsconfig.json` file to use `ts-node/register`. If you can make this better, though, go for it!

Reply:
Ah yeah, you probably do. It would really only prevent the need for "duplicate" `.js` and `.ts` tests being committed.